xtable-hudi-support/xtable-hudi-support-extensions/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache XTable (incubating)
+Copyright 2024 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
```

Comment:
Is it necessary to have a copy of the main NOTICE file? It seems they are identical.

Reply:
It's not necessary, will remove it.